shared/team-building/container.tsx

```diff
@@ -282,6 +283,59 @@ const deriveRolePickerArrowKeyFns = memoize(
   })
 )
 
+const alphabet = 'abcdefghijklmnopqrstuvwxyz'
+const isAlpha = (letter: string) => alphabet.includes(letter)
+const letterToAlphaIndex = (letter: string) => alphabet.indexOf(letter)
+
+// Returns array with 28 entries
+// 0 - "Recommendations" section
+// 1-26 - a-z sections
+// 27 - 0-9 section
+const sortAndSplitRecommendations = memoize(
+  (results: Unpacked<typeof deriveSearchResults>): Array<SearchRecSection> | null => {
+    if (!results) return results
+
+    const sections: Array<SearchRecSection> = [
+      {
+        data: [],
+        label: 'Recommendations',
+        shortcut: false,
+      },
+    ]
+    results.forEach(rec => {
+      if (!rec.contact) {
+        sections[0].data.push(rec)
+        return
+      }
+      const letter = (rec.prettyName || rec.displayLabel || '')[0].toLowerCase()
```

Comment:
`''[0]` is `undefined`, which will fail:
```
   console.log(''[0].toLowerCase());
TypeError: Cannot read property 'toLowerCase' of undefined
    at evalmachine.<anonymous>:1:19
    at Script.runInContext (vm.js:133:20)
    at Object.runInContext (vm.js:311:6)
    at evaluate (/run_dir/repl.js:133:14)
    at ReadStream.<anonymous> (/run_dir/repl.js:116:5)
    at ReadStream.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at ReadStream.Readable.push (_stream_readable.js:224:10)
    at lazyFs.read (internal/fs/streams.js:181:12)
```

Reply:
Oof, thanks